addons/vehicles/XEH_postInit.sqf

```diff
@@ -1,7 +1,38 @@
 // by esteldunedain
 #include "script_component.hpp"
 
+// eject destroyed vehicle init
+private _ejectDestroyedClasses = [];
+{
+    private _ejectDestroyedClass = configName _x;
+    if (-1 == _ejectDestroyedClasses findIf {_ejectDestroyedClass isKindOf _x}) then {
+        TRACE_1("init ejectDestroyed class",_ejectDestroyedClass);
+        _ejectDestroyedClasses pushBack _ejectDestroyedClass;
+        [_ejectDestroyedClass, "init", {
+            params ["_vehicle"];
+            if (!alive _vehicle) exitWith {};
+            TRACE_2("init ejectDestroyed vehicle",_vehicle,typeOf _vehicle);
+            _vehicle addEventHandler ["HandleDamage", {
+                params ["_vehicle", "", "", "", "_ammo"];
+                if (!alive _vehicle) then {
+                    TRACE_2("ejectDestroyed HDEH",typeOf _vehicle,_this);
+                    if (0.5 >= getNumber (configFile >> "CfgAmmo" >> _ammo >> "explosive")) then {
+                        {
+                            if (alive _x) then {
+                                moveOut _x;
```

Comment:
wasn't there a problem that moveOut didn't work with dead bodies? Or was that fixed recently?

Reply:
but I don't move out them ^^ `if (alive _x)`.
Or you want to move out them also? I don't think it's logical.

Reply:
lol duh... Must read. Read good for thinking.
I was thinking about unconscious units but this already handles that fine. Then my brain moved from unconscious to dead.